lib/modules/platform/bitbucket/index.ts

```diff
@@ -312,16 +312,18 @@ export async function findPr({
       return null;
     }
 
-    return utils.prInfo(prs[0]);
+    return utils.prInfo(prs.sort((a, b) => b.id - a.id)[0]);
   }
 
   const prList = await getPrList();
-  const pr = prList.find(
-    (p) =>
-      p.sourceBranch === branchName &&
-      (!prTitle || p.title.toUpperCase() === prTitle.toUpperCase()) &&
-      matchesState(p.state, state),
-  );
+  const pr = prList
+    .sort((a, b) => b.number - a.number)
```

Comment:
the sort at line 315  is on another branch of the code (if (includeOtherAuthors)  and handle another kind of object (PrResponse vs Pr)

Reply:
I would prefer that we sort once, in one location, instead of logic being duplicated like now and also it gets repeatedly sorted every time findPr() is called